maps/Route36.asm

```diff
@@ -179,7 +179,7 @@ TrainerSchoolboyAlan1:
 	iftrue .NumberAccepted
 	checkevent EVENT_ALAN_ASKED_FOR_PHONE_NUMBER
 	iftrue .AskAgainForPhoneNumber
-	writetext UnknownText_0x1947aa
+	writetext SchoolboyAlanBooksText
```

Comment:
Actually, do other phone trainers' scripts have a standard adjective/verb for the after-AskAgainForPhoneNumber text?

Reply:
It seems like they don't.